jnpr/openclos/trapd.py

```diff
@@ -72,8 +73,19 @@ def onTrap(transportDispatcher, transportDomain, transportAddress, wholeMsg):
                 for oid, val in varBinds:
                     logger.debug('%s = %s' % (oid.prettyPrint(), val.prettyPrint()))
                     
+    
     # start the 2-stage configuration in a separate thread
     if trapReceiver is not None:
+        # execute 2-stage configuration callback if there is one configured in openclos.yaml
```

Comment:
Since we invent this twoStageConfigurationCallback to be sort of generic, so theoretically standalone version OpenClos should be able to use it too.

Thanks.
-yun

From: moloyc [mailto:notifications@github.com]
Sent: Wednesday, February 04, 2015 11:23 PM
To: Juniper/OpenClos
Cc: Yun Li
Subject: Re: [OpenClos] added check for master in trap daemon. it can be configured in openclos.... (#147)


In jnpr/openclos/trapd.py<https://github.com/Juniper/OpenClos/pull/147#discussion_r24142536>:

>      # start the 2-stage configuration in a separate thread

>      if trapReceiver is not None:

> +        # execute 2-stage configuration callback if there is one configured in openclos.yaml

Do you want to add isNdIntegrated check here?

—
Reply to this email directly or view it on GitHub<https://github.com/Juniper/OpenClos/pull/147/files#r24142536>.